src/test/regress/expected/multi_utilities.out

```diff
@@ -446,7 +446,32 @@ SELECT 1 FROM citus_create_restore_point('regression-test');
         1
 (1 row)
 
--- TODO: support VERBOSE
--- VACUUM VERBOSE dustbunnies;
--- VACUUM (FULL, VERBOSE) dustbunnies;
--- ANALYZE VERBOSE dustbunnies;
+-- verify VERBOSE is supported
+VACUUM (FULL, VERBOSE) dustbunnies;
+INFO:  vacuuming "public.dustbunnies"
+INFO:  "dustbunnies": found 0 removable, 0 nonremovable row versions in 0 pages
+DETAIL:  0 dead row versions cannot be removed yet.
+CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s.
+INFO:  FROM localhost:57638 MESSAGE: INFO:  vacuuming "public.dustbunnies_990002"
+
+INFO:  FROM localhost:57638 MESSAGE: INFO:  "dustbunnies_990002": found 0 removable, 6 nonremovable row versions in 1 pages
+DETAIL:  0 dead row versions cannot be removed yet.
+CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s.
+
+INFO:  FROM localhost:57637 MESSAGE: INFO:  vacuuming "public.dustbunnies_990002"
+
+INFO:  FROM localhost:57637 MESSAGE: INFO:  "dustbunnies_990002": found 0 removable, 6 nonremovable row versions in 1 pages
```

Comment:
I also wonder whether we should move the hostname into CONTEXT, since that's what we do everywhere else. Or maybe not show it at all.

Reply:
AFAIK CONTEXT is invoked when there is an ERROR. We do not error out for vacuum verbose. I also think that it would be beneficial to provide it for debugging purposes if we want to use this mechanism for other notices as well. So instead of not showing, and putting it into context, I want to put it into Detail. Does this make sense?